docs/source/testing.rst

```diff
@@ -47,16 +47,16 @@ An (almost) exhaustive example of test mode:
   ...             "issue_uri": "https://github.com/osuosl/ganeti_webmgr/issues",
   ...             "date_worked": "2014-04-17"
   ...         }
-  >>> ts.create_time(params)
+  >>> ts.create_time(parameter_dict=params)
```

Comment:
In this case, I don't think the keyword is necessary, since there is only one argument, and it contains kay:value pairs, also parameter_dict doesn't have any real meaning that helps understanding